test/package.json

```diff
@@ -68,7 +68,7 @@
     "file-type": "17.1.6",
     "http-status": "1.6.2",
     "jwt-decode": "4.0.0",
-    "lexical": "0.15.0",
+    "lexical": "0.16.1",
```

Comment:
Any notable changes on this pre 1.0 bump?

Reply:
This was already bumped from 0.15.0 to 0.16.1 in a previous PR few weeks ago. We just forgot to update that package in the test directory as well